lib/assessment.md

```diff
@@ -42,6 +52,10 @@ What selectors could we write to reference each of the following categories of e
 
 <!-- your answer starts here -->
 
+ * <div class="big"> elements </div>
+ * <div id="contentPane"> </div>
+ * <div class="important"> <div class="red"> </div> </div>  <!-- not 100% sure -->
+ * <div> <p> </p> </div>
```

Comment:
I think you've misunderstood what this question is asking. You were directed to give the _CSS selector_ which referred to the element or group of elements described.